evm_loader/program/src/config.rs

```diff
@@ -0,0 +1,183 @@
+//! CONFIG MODULE
+use cfg_if::cfg_if;
+
+use evm::{ U256 };
+
+cfg_if! {
+    if #[cfg(feature = "mainnet")] {
+
+        const CHAIN_ID: u64 = 111;
+        /// `PAYMENT_TO_COLLATERAL_POOL`
+        pub const PAYMENT_TO_COLLATERAL_POOL: u64 = 1000;
+        /// `PAYMENT_TO_DEPOSIT`
+        pub const PAYMENT_TO_DEPOSIT: u64 = 1000;
+        /// `OPERATOR_PRIORITY_SLOTS`
+        pub const OPERATOR_PRIORITY_SLOTS: u64 = 16;
+
+        macros::pubkey_array!(
+            AUTHORIZED_OPERATOR_LIST,
+            [
+                "9kPRbbwKL5SYELF4cZqWWFmP88QkKys51DoaUBx8eK73",
+                "BMp6gEnveANdvSvspESJUrNczuHz1GF5UQKjVLCkAZih",
+                "CVAimMqtcmSUCV4RLZSJAreDpEd7JEZmrvCVj85yaRzZ",
+            ]
+        );
+
+        /// Token Mint ID
+        pub mod token_mint {
+            use crate::macrorules::{ str_as_bytes_len, neon_elf_param, declare_param_id };
+
+            declare_param_id!(NEON_TOKEN_MINT, "HPsV9Deocecw3GeZv1FkAPNCBRfuVyfw9MMwjwRe1xaU");
+            /// Ethereum account version
+            pub const DECIMALS: u8 = 9;
+
+            /// Number of base 10 digits to the right of the decimal place
+            #[must_use]
+            pub const fn decimals() -> u8 { DECIMALS }
+        }
+
+        /// Collateral pool base address
+        pub mod collateral_pool_base {
+            use crate::macrorules::{ str_as_bytes_len, neon_elf_param, declare_param_id };
+
+            declare_param_id!(NEON_POOL_BASE, "4sW3SZDJB7qXUyCYKA7pFL8eCTfm3REr8oSiKkww7MaT");
+
+            /// `COLLATERAL_SEED_PREFIX`
+            pub const PREFIX: &str = "collateral_seed_";
+        }
+
+    } else if #[cfg(feature = "testnet")] {
+
+        const CHAIN_ID: u64 = 111;
+        /// `PAYMENT_TO_COLLATERAL_POOL`
+        pub const PAYMENT_TO_COLLATERAL_POOL: u64 = 1000;
+        /// `PAYMENT_TO_DEPOSIT`
+        pub const PAYMENT_TO_DEPOSIT: u64 = 1000;
+        /// `OPERATOR_PRIORITY_SLOTS`
+        pub const OPERATOR_PRIORITY_SLOTS: u64 = 16;
+
+        macros::pubkey_array!(
+            AUTHORIZED_OPERATOR_LIST,
+            [
+                "9kPRbbwKL5SYELF4cZqWWFmP88QkKys51DoaUBx8eK73",
+                "BMp6gEnveANdvSvspESJUrNczuHz1GF5UQKjVLCkAZih",
+                "CVAimMqtcmSUCV4RLZSJAreDpEd7JEZmrvCVj85yaRzZ",
+            ]
+        );
+
+        /// Token Mint ID
+        pub mod token_mint {
+            use crate::macrorules::{ str_as_bytes_len, neon_elf_param, declare_param_id };
+
+            declare_param_id!(NEON_TOKEN_MINT, "HPsV9Deocecw3GeZv1FkAPNCBRfuVyfw9MMwjwRe1xaU");
```

Comment:
For `testnet` and `devnet` use following values:
NEON_TOKEN_MINT: 89dre8rZjLNft7HoupGiyxu3MNftR577ZYu8bHe2kK7g
NEON_POOL_BASE: 7SBdHNeF9FFYySEoszpjZXXQsAiwa5Lzpsz6nUJWusEx

For `mainnet` we will change it later.